### DISCUSSION TITLE
doris1.0.0监控使用哪个版本的dashboard

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#11239] [A - General / Q&A] [answered]
doris1.0.0发现原来的 dashboard revision3 不能用了，发现最新的版本是revision4，就试着升级到revision4，但是发现dashboard解析到的数据有问题

**Comment by JNSimba** [comment]
What does a problem with data parsing mean?

**Comment by stgztsw** [comment, marked as answer]
不好意思，无视这个问题，监控看板没有问题
